### PR TITLE
Bosh GET method

### PR DESCRIPTION
**Pull request by michalwski**
The GET method was implement only for Load Balancer's health checks. For example ELB uses this method.

### CODE REVIEW

File: test/ejabberd_tests/tests/bosh_SUITE.erl [hunk 1]
```diff
@@ -59,9 +59,13 @@ suite() ->
     escalus:suite().
 
 essential_test_cases() ->
-    [create_and_terminate_session,
-     accept_higher_hold_value,
-     do_not_accept_0_hold_value].
+    [%create_and_terminate_session,
```
Comment (lavrin) [inline test/ejabberd_tests/tests/bosh_SUITE.erl]: This _seems_ to be accidental... Should we actually comment them out?

Comment (michalwski) [inline test/ejabberd_tests/tests/bosh_SUITE.erl]: Correct, it's accidental.

File: apps/ejabberd/src/mod_bosh.erl [hunk 2]
```diff
@@ -190,6 +192,14 @@ info(accept_options, Req, State) ->
     Headers = ac_all(Origin),
     ?DEBUG("OPTIONS response: ~p~n", [Headers]),
     {ok, strip_ok(cowboy_req:reply(200, Headers, <<>>, Req2)), State};
+info(accept_get, Req, State) ->
```
Comment (fenek) [inline apps/ejabberd/src/mod_bosh.erl]: `accept_get` is missing in `info()` type.

Comment (michalwski) [inline apps/ejabberd/src/mod_bosh.erl]: good catch :)

**Pull request merged**